flask_project/config.py

```diff
@@ -15,7 +20,22 @@ def get_ipqs_secret_key():
     return ipqs_secret_key
 
 
+def get_database_uri():
+    """
+    Retrieve database_url from the .env. If not exception if raised.
+    Returns:
+        - str: database_url.
+    """
+    sqlalchemy_database_uri = os.environ.get("DATABASE_URL")
+    if not sqlalchemy_database_uri:
+        raise Exception("No database connection!")
```

Comment:
I think the message should be replaced with "No database configuration found" or something similar.